### PR TITLE
feat(icon): added multicluster icon

### LINKED ISSUE #5451: NEW ICON - multicluster
Per https://github.com/patternfly/patternfly-design/issues/1237 request, new icon for multicluster is created. Here is the svg. file
multicluster

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #5451

### CODE REVIEW

File: scripts/iconList.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-ICON_SRC='./src/icons/definitions/pf-icons.json'
+ICON_SRC='./src/icons/definitions/pf-icons.mjs'
```
Comment (mcoker) [inline scripts/iconList.sh:3]: 👍

**Review by mcoker — APPROVED** [review]
LGTM!

**Comment by mcoker** [comment]
cc @yichen1yu @mcarrano

**Review by mcarrano — APPROVED** [review]
Looks good to me!

**Comment by mcoker** [comment]
@jenny-s51 you should be good to rebase now that https://github.com/patternfly/patternfly/pull/5538 has merged. Though it may be easier just to apply the changes again from scratch? FWIW `pf-icons.mjs` moved to https://github.com/patternfly/patternfly/blob/v5/src/icons/definitions/pficons.mjs

**Review by jenny-s51 — COMMENTED** [review]
Thanks @mcoker - applied these changes again

**Review by mcoker — APPROVED** [review]
🎯

**Pull request merged**